src/lambda/models/samLambdaRuntime.ts

```diff
@@ -13,50 +13,27 @@ export type SamLambdaRuntime =
     'python3.7' |
     'python3.6' |
     'python2.7' |
-    'python' |
     'nodejs6.10' |
     'nodejs8.10' |
-    'nodejs' |
     'dotnetcore2.1' |
     'dotnetcore2.0' |
-    'dotnetcore1.0' |
-    'dotnetcore' |
-    'dotnet' |
-    'go1.x' |
-    'go' |
-    'java8' |
-    'java' |
-    'ruby' |
-    'ruby2.5'
+    'dotnetcore1.0'
```

Comment:
Unfortunately, debugging is only supported for .NET Core 2.x.

Reply:
Purged